README.md

```diff
@@ -28,3 +28,20 @@ for instructions on how to make changes.
 ## Community
 Check out our [community page](http://forsetisecurity.org/community/) for ways
 to engage with the Forseti Community.
+
+## Integration test
+To run integration tests on your local environment without having to 
+store sensitive information/secrets in the forseti-security repository, store
+project id in TF_VAR_project_id variable, organization id in TF_VAR_org_id, 
+domain in TF_VAR_domain, billing account in TF_VAR_billing_account and 
+service account key in SERVICE_ACCOUNT_JSON variable.
```

Comment:
This is exactly what I meant by `setting environment variables from bash shell`.  Users/developers can go:
`export TF_VAR_project_id=<PROJECT_ID>`

Reply:
This way secrets need not be stored in the `forseti-security` repo as the environment variables can be set from the bash shell.

Reply:
Currently the tests can only be run on the silver org. Users would need the service account key from this org in order to run the tests locally. I think we should hold off on instructing the community that they can run integration tests until these tests are supported in any environment.

Reply:
The command below shows how to run tests from the color domain so that developer can run tests on their local testing without storing secrets in `forseti-security` repository.

Reply:
Sorry, I was not clear. The tests can be run against other orgs but the tests will fail because there are several tests that check for resources that only exist in the silver org, such as: https://github.com/forseti-security/forseti-security/blob/master/integration_tests/tests/forseti/controls/explain.rb#L47. I think we should close this PR and I will work on updating any of these tests to be flexible.

Reply:
Inventory and model modules will not be impacted. Tests in Explain that are subject to Silver domain will fail as all the tests were written to run in Silver domain only.